sretoolbox/container/image.py

```diff
@@ -47,6 +54,11 @@ def __init__(self, url, tag_override=None, username=None, password=None,
         else:
             self.tag = tag_override
 
+        self._cache_digest = None
+        # If the URL was by-digest, we can cache this right away.
+        if image_data['digest']:
+            self._cache_digest = f"sha256:{image_data['digest']}"
```

Comment:
the `sha256:` should be coming from the `image_data['digest']`... more on that ahead

Reply:
Ack, I’m on board.